volontulo/migrations/0001_initial.py

```diff
@@ -47,27 +49,27 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='UserBadges',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
-                ('created_at', models.DateTimeField(blank=True, default=django.utils.timezone.now)),
+                ('id', models.AutoField(serialize=False, auto_created=True, primary_key=True, verbose_name='ID')),
+                ('created_at', models.DateTimeField(default=django.utils.timezone.now, blank=True)),
                 ('description', models.CharField(max_length=255)),
                 ('badge', models.ForeignKey(to='volontulo.Badge')),
+                ('content_type', models.ForeignKey(null=True, to='contenttypes.ContentType')),
             ],
         ),
         migrations.CreateModel(
             name='UserProfile',
             fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
-                ('is_organization', models.BooleanField(default=False)),
+                ('id', models.AutoField(serialize=False, auto_created=True, primary_key=True, verbose_name='ID')),
                 ('is_administrator', models.BooleanField(default=False)),
                 ('badges', models.ManyToManyField(related_name='user_profile', to='volontulo.Badge', through='volontulo.UserBadges')),
-                ('organization', models.ForeignKey(to='volontulo.Organization', blank=True, null=True)),
+                ('organization', models.ForeignKey(null=True, blank=True, to='volontulo.Organization')),
                 ('user', models.OneToOneField(to=settings.AUTH_USER_MODEL)),
             ],
         ),
         migrations.AddField(
             model_name='userbadges',
-            name='user',
-            field=models.ForeignKey(to='volontulo.UserProfile'),
+            name='userprofile',
+            field=models.ForeignKey(db_column='userprofile_id', to='volontulo.UserProfile'),
         ),
```

Comment:
I'm afraid of altering already existing migration files. Is this change neccessary in this particular file?